app/main.py

```diff
@@ -1,3 +1,6 @@
 def count_occurrences(phrase: str, letter: str) -> int:
-    # write your code here
-    pass
+    counter = 0
+    for char in phrase:
+        if char.lower() == letter.lower():
+            counter += 1
+    return counter
```

Comment:
It is recommended to avoid using loops to solve this task. Instead, consider utilizing Python's built-in string functions to simplify the solution